script/check_chromium_src.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+#
+# This script looks for problems in chromium_src.
+#
+# 1. Look for files in chromium_src that don't have a corresponding
+#    file under src tree (or grit).
+#
+# 2. Look for #defines that redefine symbols that aren't present in
+#    the overriden file.
+#
+# 3. Look for #include statements of the overriden file that have
+#    incorrect number of ../ in the path.
+#
+# !!! This script does return false positives, but better check a
+# !!! few of those than not check at all.
+
+import argparse
+import re
+import os
+import sys
+
+"""
+Look for potential problems in chromium_src overrides.
+"""
+
+BRAVE_SRC = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+BRAVE_CHROMIUM_SRC = os.path.join(BRAVE_SRC, 'chromium_src')
+CHROMIUM_SRC = os.path.abspath(os.path.dirname(BRAVE_SRC))
+
+
+# This list of EXCLUDES will only work as expected when this script is run
+# from UNIX-like shells (e.g. Linux, Mac or shells like Git Bash on Windows).
+EXCLUDES = [
+  'CPPLINT.cfg',
+  '_(unit|browser)test(_mac)?.cc',
+  'third_party/blink/renderer/modules/battery/navigator_batterytest.cc',
+  'third_party/blink/renderer/modules/bluetooth/navigator_bluetoothtest.cc',
+  'third_party/blink/renderer/modules/quota/navigator_storagetest.cc',
+  'third_party/blink/renderer/modules/storage/brave_dom_window_storage.h',
+  'chrome/installer/linux/common/brave-browser/chromium-browser.appdata.xml',
+  'chrome/installer/linux/common/brave-browser/chromium-browser.info',
```

Comment:
These do show up in the output on Windows.

Reply:
Yes, this is what we discussed on Slack the other day, and the reason of the following comment in the code:
```
# This list of EXCLUDES will only work as expected when this script is run
# from UNIX-like shells (e.g. Linux, Mac or shells like Git Bash on Windows).
```
Basically, it should work fine if you run it from something like Git Bash or WSL, but not from PowerShell. It does bother me a bit though, I'll try to come up with a solution that works well everywhere.

Reply:
@mkarolin Just replaced the last commit with a new one that makes this work identically everywhere (tested on Windows' cmd.exe, git Bash and WSL). Please take another look, thanks!

Reply:
Works as expected now. I did test it with `git Bash` before (and now), but I think the issues wasn't the shell itself, but Python. Since `git Bash` doesn't come with a built in Python, my regular Windows Python install is used and it does use the back slashes.

Reply:
Ah, I see. Thanks for clarifying! Merging now then